Source/InformationObjectDefinition/gdcmIODEntry.h

```diff
@@ -64,7 +64,7 @@ class GDCM_EXPORT IODEntry
   void SetRef(const char *ref) { Ref = ref; }
   const char *GetRef() const { return Ref.c_str(); }
 
-  void SetUsage(const char *usag) { usage = usag; }
+  void SetUsage(const char *usage) { usage = usage; }
```

Comment:
This looks a little strange. Might work, but would probably cause a _-Wshadow_ warning if enabled.

Reply:
Ah true, will fix. Thanks.

Reply:
Fixed